Modules/_collectionsmodule.c

```diff
@@ -847,16 +877,30 @@ deque_inplace_repeat(dequeobject *deque, Py_ssize_t n)
     return (PyObject *)deque;
 }
 
+static PyObject *
+deque_inplace_repeat(dequeobject *deque, Py_ssize_t n)
+{
+    PyObject *result = NULL;
```

Comment:
My inclination is that we should prefer not defensively initializing variables where it's not necessary.

```suggestion
    PyObject *result;
```

Reply:
I don't feel super strongly about this, since the compiler should warn us, but in my experience this can help accelerate debugging.